app/views/items/browse_facet.html.erb

```diff
@@ -46,24 +46,28 @@
       <% if @res.key?(facet_name) %>
         <table class="index_table">
           <tbody>
-            <% @res[facet_name].each do |list| %>
+            <% @res[facet_name].each do |key, value| %>
+              <%# NOTE the below line accommodates Apium v1.0 as well as
+                  v1.1 when facet responses included more information %>
+              <% key, normalized, num = format_facet_response(key, value) %>
+
               <tr>
-                  <% text = list[0].blank? ? t("browse.no_label", default: "No label") : value_label(facet_name, list[0]) %>
+                  <% text = normalized.blank? ? t("browse.no_label", default: "No label") : value_label(facet_name, normalized) %>
                   <td class="index_num">
                     <%= link_to prefix_path("search_path",
-                      f: ["#{facet_name}|#{list[0]}"]), rel: "nofollow" do %>
-                      <%= list[1] %>
+                      f: ["#{facet_name}|#{key}"]), rel: "nofollow" do %>
+                      <%= num %>
                     <% end %>
                   </td>
                   <td class="index_spacer">
                     <%= link_to prefix_path("search_path",
-                      f: ["#{facet_name}|#{list[0]}"]), rel: "nofollow" do %>
+                      f: ["#{facet_name}|#{key}"]), rel: "nofollow" do %>
                       <span></span>
                     <% end %>
                   </td>
                   <td class="index_name">
                     <%= link_to prefix_path("search_path",
-                      f: ["#{facet_name}|#{list[0]}"]), rel: "nofollow" do %>
+                      f: ["#{facet_name}|#{key}"]), rel: "nofollow" do %>
```

Comment:
:thinking: So `format_facet_response` is supposed to return `[normalized key, label for display, num]`. This makes the assignment of its responses to `key, normalized, num` confusing. Thinking the first should either remain `key` or be switched to `normalized`. and the second return value should be `label`?

The replacement of `list[0]` by `key` in some places and `normalized` in others led me to this.

(edit)
Okay, so this was addressed in a follow-up commit by renaming `normalized` to `nonnormalized`, but it still looks a little odd in other method calls where I think it might be clearer if the variables match the names of the kwargs used later. `nonnormalized` alone is an adjective without a noun and harder to understand without reviewing more code for context.

e.g.
```ruby
                facet_label(
                  type: facet_name,
                  normalized: key,
                  label: nonnormalized
```
could be
```ruby
                facet_label(
                  type: facet_name,
                  normalized: key,
                  label: label
```
or similar.

Reply:
Yeah, I was struggling a lot with how to name things, considering that if you're using an older API then nothing is normalized, technically, whereas in the newer API (and going forward) they are normalized.  I don't remember why I didn't end up using "label" even though I kind of wanted to, like perhaps somewhere in existing code that was already being used?  I can revisit this and see if I can come up with something better.